qhexview.cpp

```diff
@@ -626,6 +683,33 @@ int64_t QHexView::pixelToWord(int x, int y) const {
 	return word;
 }
 
+//------------------------------------------------------------------------------
+// Name: updateToolTip
+//------------------------------------------------------------------------------
+void QHexView::updateToolTip() {
+	if(selectedBytesSize() <= 0) {
+		return;
+	}
+
+	auto sb = selectedBytes();
+	auto tooltip = QString("<p style='white-space:pre'>"); //prevent word wrap
+#if 0
+	// Plain formatting with 0x prefix
+	tooltip += QString("<b>Addr:</b> 0x%1 - 0x%2").arg(selectedBytesAddress(), 0, 16)
+												  .arg(selectedBytesAddress() + sb.size(), 0, 16);
+#else
+	// Segment:Offset style formatting is easier to read
+	const address_t start = selectedBytesAddress();
+	const address_t end = selectedBytesAddress() + sb.size();
+	tooltip += "<b>Addr: </b>" + formatAddress(start) + " - " + formatAddress(end);
```

Comment:
I think you want to use `QStringBuilder` when concating many strings together. The performance impact of a tooltip string render being inefficient might be extremely unnoticeable, but I like having other written code be good reference material.